maestro/audit.py

```diff
@@ -56,6 +57,12 @@ def _format_what_compact(self, what):
         return '{} container{}'.format(
             len(what), 's' if len(what) > 1 else '')
 
+    def _format_what_services_ships(self, what):
```

Comment:
If you're only using this is the `ExecuteScriptAuditor` then it doesn't belong in the `BaseAuditor`. If other auditors need it later, we can always move it then.